openlibrary/plugins/openlibrary/js/modals/index.js

```diff
@@ -340,13 +340,13 @@ function clearForm($form) {
  * @param {String} modalId  A string that uniquely identifies a modal.
  * @param {String} [reloadId]   ID of list receiving a reload event
  */
-function displayModal(modalId, reloadId) {
+function displayModal(modalId, reloadId, maxWidth) {
```

Comment:
This is an incredibly bad way of doing this (passing `maxWidth` through `addClickListeners`).  Once the dust settles a bit, I plan on splitting this file into two separate files: one for notes functionality and one for tags functionality.  I suspect that we soon will not have a notes modal at all, anyway.

I intend on updating all of this after version one, if there are no objections.

Reply:
Another option would be to add a `data-` attribute to the modal specifying their max-width